docs/querying/basics.md

```diff
@@ -242,7 +242,16 @@ These 2 queries will produce the same result.
 
 This modifier is disabled by default since it breaks the invariant that PromQL
 does not look ahead of the evaluation time for samples. It can be enabled by setting
-`--enable-feature=promql-at-modifier` flag. It will be enabled by default in the future.
+`--enable-feature=promql-at-modifier` flag. See [disabled features](../disabled_features.md) for more about this flag.
+
+Additionally, `start()` and `end()` can also be used as values for the `@` modifier.
+
+For a range query, their values are the start and end of the range query respectively and remain the same for all steps.
```

Comment:
"their values are" → "they resolve to" (to be consistent to the next sentence, and to not confuse with too many uses of the word "value").